trigger_data.py

```diff
@@ -58,42 +58,46 @@
     input_fifo = os.open(input_path, os.O_RDONLY)
     output_fifo = os.open(output_path, os.O_WRONLY)
 
-    ser = Serial(oscope_path, 1500000, timeout=1.0)
-
-    def handler(signum, frame):
-        os.close(input_fifo)
-        os.close(output_fifo)
-        ser.close()
+    if verbose:
+        print("reading")
+    result = os.read(input_fifo, 1)
+    if len(result) != 1:
         exit(1)
 
-    signal.signal(signal.SIGINT, handler)
-
-    while True:
+    if verbose:
+        print(unpack('%dB' % len(result), result))
+    if result == b'1':
```

Comment:
what if one byte is read but it's not `\x01`, like `EOF` or something? might just want a `syslog` in that case.

Reply:
Yeah we can do that. The output from the awk is actually `0x49 0x0a 0x48 0x0a` which is 1\n0\n